src/checkqueue.h

```diff
@@ -113,11 +113,8 @@ class CCheckQueue
                 // * Try to account for idle jobs which will instantly start helping.
                 // * Don't do batches smaller than 1 (duh), or larger than nBatchSize.
                 nNow = std::max(1U, std::min(nBatchSize, (unsigned int)queue.size() / (nTotal + nIdle + 1)));
-                vChecks.resize(nNow);
```

Comment:
9d454e8447456c91ec45512d0caaa71bd0bb8c0f: Changing `resize` into `push_back` (with implicit `resize`) seems like a unrelated behaviour change that violates the comment you removed?

Reply:
@MarcoFalke Does your comment consider that the required memory has been pre-allocated already: https://github.com/bitcoin/bitcoin/blob/cb552c5f21192d2f8f69c07130928c21301a5e7f/src/checkqueue.h#L73?